aspnetcore/blazor/events.md

```diff
@@ -0,0 +1,249 @@
+---
+title: ASP.NET Core Blazor events
```

Comment:
Doing this we should also think about SEO.
Remember, that we have a big goal of making all these doc articles more discoverable. From that perspective, I'm not sure whether people would use `handling` as a phrase to look for in title vs only `Events`. @danroth27 what do you think about this one: `Events` vs `Event Handling`

Reply:
That's cool if you feel it best. The docs guidelines only ask for it where possible. For things that are actually gerunds, it's cool with them (e.g., "logging," "data binding," etc.).

Reply:
**_Done!_** :+1: